packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -3424,13 +3425,15 @@ export { Notification_2 as Notification }
 
 // @public
 export interface NotificationProps {
+    ariaLive?: 'assertive' | 'off' | 'polite';
     autoDismiss?: boolean;
     notificationIconProps?: IIconProps;
     notificationStrings?: NotificationStrings;
     onClickLink?: () => void;
     onClickPrimaryButton?: () => void;
     onClickSecondaryButton?: () => void;
     onDismiss?: () => void;
+    role?: string;
```

Comment:
Do we want to restrict this type to only be for `'alert' | 'status'`?

Reply:
Good call! updated